### DISCUSSION TITLE
Application for cmi

### DISCUSSION THREAD

**Discussion by livxbox5** [Zrips/CMI#7591] [Feature Requests]
make an npc module for creating npc and their settings

**Comment by CloudeLecaw** [comment]
what this module should do in your eyes?
by spawnmob command i could spawn all types of mobs with all attributes that vanilla offers.. so what settings you mean?

**Reply by livxbox5** [reply]
not the type of Citizens plugin Creating an NPC is only slender in the CMI plugin in the module

**Comment by CloudeLecaw** [comment]
well i thing doing anything npc related is out of the scope of cmi
let Citizens and co do the job, they do a great one allready
